pandas/tests/resample/test_period_index.py

```diff
@@ -134,7 +134,7 @@ def test_basic_downsample(self, simple_period_range_series):
         "rule,expected_error_msg",
         [
             ("y-dec", "<YearEnd: month=12>"),
-            ("qe-mar", "<QuarterEnd: startingMonth=3>"),
+            ("Q-MAR", "<QuarterEnd: startingMonth=3>"),
```

Comment:
nice - this is a period, so indeed the period alias should be used

Reply:
yeah, sorry I missed it PR #55553